src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,82 @@
+import { FC, useState } from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const [selectedPersonId, setSelectedPersonId] = useState('');
+  const getPersonByName = (name: string): Person | undefined => {
+    return people.find(person => person.name === name);
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const mother = getPersonByName(person.motherName || '-');
+          const father = getPersonByName(person.fatherName || '-');
+
+          return (
+            <tr
+              data-cy="person"
+              key={person.slug}
+              className={
+                selectedPersonId === person.slug
```

Comment:
Not quite sure, but maybe test  failing occurs because it wants you to use 'ClassNames' import (IMHO)

Reply:
Hmmm. Nice idea, I will try, thank you